wolfssl/ssl.h

```diff
@@ -3834,6 +3834,7 @@ WOLFSSL_API int wolfSSL_set1_curves_list(WOLFSSL* ssl, const char* names);
     defined(HAVE_STUNNEL) || defined(WOLFSSL_MYSQL_COMPATIBLE) || \
     defined(WOLFSSL_NGINX) || defined(WOLFSSL_HAPROXY)
 
+WOLFSSL_API int wolfSSL_get_verify_mode(WOLFSSL* ssl);
```

Comment:
OpenSSL takes in a `const` parameter. Omitting this could generate warnings. Probably best to use the same API. `wolfSSL_CTX_get_verify_mode` could also be changed.

Reply:
I saw that but noticed that `wolfSSL_get_verify_depth` and `wolfSSL_get_verify_callback` also were missing the `const`. I feel like that is starting to creep to a much larger change. I'd prefer to have a separate PR to address that and maybe other constness compatible calls.

Thoughts?

Reply:
I personally prefer fewer PR's since each one has some overhead attached to it so I try to include any small changes I see in a relevant big PR. I don't think that this is becoming that big of PR so I think that the `const` change could be easily fit in this PR.

Reply:
ok added `const` for the `*_get_verify_mode` but not trying to address other differences from openSSL